app/views/shareable_links/_my_module_details.html.erb

```diff
@@ -0,0 +1,58 @@
+<% my_module_editable = false %>
+
+<div class="flex-block date-block">
+  <div class="flex-block-label">
+    <span class="sn-icon block-icon sn-icon-calendar mr-2.5"></span>
+    <span class="hidden-xs hidden-sm hidden-md"><%= t('my_modules.details.start_date') %></span>
+  </div>
+  <div id="startDateContainer" class="datetime-container">
+    <span class="date-text">
+      <span id="startDateLabelContainer" class="view-block">
+        <%= render partial: "my_modules/start_date_label.html.erb" , locals: { my_module: @my_module, start_date_editable: false } %>
```

Comment:
remove start_date_ediatable, bacuse my_modules/start_date_label does not use it anymore